README.md

```diff
@@ -230,6 +230,7 @@ export default connectAsync({ loadDataAsProps })(BookList);
 #### `loadCollection({ resource, id, opts, forceFetch })`
 #### `createResource({ resource, id, opts })`
 #### `updateResource({ resource, id, opts })`
+#### `updateCollection({ resource, id, opts })`
```

Comment:
We need an explicit breakdown of how `opts.method` can override default `method` in README.